smart-contracts/hardhat.config.js

```diff
@@ -166,6 +104,7 @@ module.exports = {
   },
   solidity: {
     compilers: [
+      { version: '0.4.18', settings },
```

Comment:
Mind providing a bit of clarity about why this got added?

Reply:
oh my bad... removed. I used it to compile WETH but found out the ABI was in `hardly-difficult-eth` package